README.md

```diff
@@ -1,135 +1,73 @@
+# Apache Airavata
+
 [![Build Status](https://travis-ci.org/apache/airavata.svg?branch=master)](https://travis-ci.org/apache/airavata)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.apache.airavata/airavata/badge.svg)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.apache.airavata%22)
+
+## About
+
+Apache Airavata is a software framework for executing and managing computational
+jobs on distributed computing resources including local clusters,
+supercomputers, national grids, academic and commercial clouds. Airavata builds
+on general concepts of service oriented computing, distributed messaging, and
+workflow composition and orchestration. Airavata bundles a server package with
+an API, client software development Kits and a general purpose reference UI
+implementation -
+[Apache Airavata Django Portal](https://github.com/apache/airavata-django-portal).
+
+Learn more about Airavata at
+[https://airavata.apache.org](https://airavata.apache.org).
+
+## Building Apache Airavata
+
+### Prerequisites
+
+- Sources compilation require Java SDK 8 or higher.
```

Comment:
For master branch, we are still stuck for java 8. It doesn't work for later version

Reply:
I guess we should tag the master branch to a release branch and merge develop to master soon.

Reply:
+1

Reply:
Good catch @DImuthuUpe . I'll update the version to just 8 then when this is merged back to develop we can update it to "11 or higher".